src/main/java/htsjdk/samtools/SamFileValidator.java

```diff
@@ -138,6 +140,14 @@ public void setIgnoreWarnings(final boolean ignoreWarnings) {
     }
 
     /**
+     * Sets whether or not we should go ahead and run mate validation beyond the mate cigar check.  For extreme cases,
+     * mate validation can require a ton of memory and we want to provide the option of skipping it.
+     */
+    public void setSkipMateValidation(final boolean skipMateValidation) {
```

Comment:
You should probably add a matching getter, someone always ends up wanting one for some reason or another.

Reply:
Can you reword the comment?  It reads like an email, not javadoc.